hawk_test_ssh.py

```diff
@@ -13,33 +13,26 @@ def __init__(self, hostname, secret=None):
         self.ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy)
         self.ssh.connect(hostname=hostname.lower(), username="root", password=secret)
 
-    def check_cluster_conf_ssh(self, command, mustmatch):
+    def check_cluster_conf_ssh(self, command):
+        '''
+        run command remotely. Return bool and command output
+        '''
         _, out, err = self.ssh.exec_command(command)
         out, err = map(lambda f: f.read().decode().rstrip('\n'), (out, err))
         print(f"INFO: ssh command [{command}] got output [{out}] and error [{err}]")
         if err:
             print(f"ERROR: got an error over SSH: [{err}]")
-            return False
-        if isinstance(mustmatch, str):
-            if mustmatch:
-                if mustmatch in out:
-                    return True
-                return False
-            return out == mustmatch
-        if isinstance(mustmatch, list):
-            for exp in mustmatch:
-                if exp not in out:
-                    return False
-            return True
-        raise ValueError("check_cluster_conf_ssh: mustmatch must be str or list")
+            return False, ''
+        return True, out
 
     @staticmethod
     def set_test_status(results, test, status):
         results.set_test_status(test, status)
 
     def verify_stonith_in_maintenance(self, results):
         print("TEST: verify_stonith_in_maintenance")
-        if self.check_cluster_conf_ssh("crm status | grep stonith-sbd", ["unmanaged", "maintenance"]):
+        ret, out = self.check_cluster_conf_ssh("crm status | grep stonith-sbd")
+        if ret and any([m in out for m in ["unmanaged", "maintenance"]]):
```

Comment:
new pylint warning
hawk_test_ssh.py:35:19: R1729: Use a generator instead 'any(m in out for m in ['unmanaged', 'maintenance'])' (use-a-generator)